source/triggers/clauses/clauseProto_OnEvent.js

```diff
@@ -71,7 +71,7 @@ this.reset = function() {
 
 this.evaluateClause = function() {
     var retVal = ( this.lastEventTime$ > 0 ) && 
-                 ( Date.now() - this.lastEventTime$ <= this.threshold$ );
+                 ( this.time - this.lastEventTime$ <= this.threshold$ );
```

Comment:
Oh wait... do I have those two functions backwards? 

We want to use replicated time here.  The same may be true of my comment above.